app/js/lib/foxbox/settings.js

```diff
@@ -33,12 +33,19 @@ const storage = localStorage ? localStorage : {
 
 export default class Settings extends Model {
   constructor() {
+    let localHostName = storage.getItem(`${PREFIX}localHostname`) ||
+      DEFAULT_HOSTNAME;
+
+    let pollingEnabled = storage.getItem(`${PREFIX}pollingEnabled`) !== null ?
```

Comment:
you can probably safely use `const` here as the scope is limited to the `constructor()` method.